arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -3496,16 +3609,7 @@
           "description": "Name of the connector."
         },
         "connectorType": {
-          "type": "string",
-          "enum": [
-            "CRM",
-            "AzureBlob",
-            "Salesforce"
-          ],
-          "x-ms-enum": {
-            "name": "ConnectorTypes",
-            "modelAsString": false
```

Comment:
ah, I was thinking about generated SDKs and such! (I'm not involved in that so I don't know if/what you have in that regard)